src/backend/distributed/worker/worker_shard_visibility.c

```diff
@@ -460,8 +460,19 @@ FilterShardsFromPgclass(Node *node, void *context)
 			MemoryContext originalContext = MemoryContextSwitchTo(queryContext);
 
 			/* add NOT relation_is_a_known_shard(oid) to the security quals of the RTE */
-			rangeTableEntry->securityQuals =
```

Comment:
Comment in this function still talks about security quals.

Reply:
https://github.com/citusdata/citus/pull/7577/files#diff-85383f96eb6780b49b1807c6e1c022b96f5c41999fb06c4385ec2e477fa1b57eR425

This still talks about security quals ...